index.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio Website</title>
+    <link rel="stylesheet" href="/styles.css">
+</head>
+
+<body>
+    <header>
+        <div class="logo">John Doe</div>
+        <div>
+            <img class="nav-icon" src="/Icons/Enabled.png" alt="nav button">
+        </div>
+    </header>
+    <main>
+        <section>
+            <h1>
+                Hey There.<br>I'm Alex
+            </h1>
+            <h6>I'am a Software Developer</h6>
+            <p>I can help you build a product , feature or website Look through some of my work and experience! If you
+                like what you see and have a project you need coded,
+                don’t hestiate to contact me.</p>
```

Comment:
- [ ] It would be nice if you could use the right font specified in the Figma design. It is best to always go with the design specifications given in order to match your design with that.
- You can do that by importing the Poppins google font and giving your `body` tag the `font-family: 'Poppins', san-serif;`. That way, your entire website will have that font as the default font. What do you think 👍

Reply:
Ok i will do that